scripts/patch-release-for-pr.js

```diff
@@ -109,8 +225,24 @@ async function main(args) {
   }
 
   // Create new branch, apply changes from all commits on PR branch, commit, push
-  const branchName = `patch-release-pr-${prNumbers.join('-')}`;
-  await run('git', 'checkout', '-b', branchName);
+  // Allow fixed branch name via environment variable for CI workflows
+  const branchName =
+    process.env.PATCH_RELEASE_BRANCH ||
+    `patch-release-pr-${prNumbers.join('-')}`;
+
+  // Check if branch already exists (for CI workflows using fixed branch name)
+  try {
+    await run(
+      'git',
+      'show-ref',
+      '--verify',
+      '--quiet',
+      `refs/heads/${branchName}`,
+    );
+    await run('git', 'checkout', branchName);
```

Comment:
it SHOULD never be a problem ... but it's tempting to suggest a pull here

Reply:
There's a `git fetch` at the top that covers it